src/handlebars/helpers.ts

```diff
@@ -14,9 +14,23 @@ export function registerHandlebarsHelpers() {
     }
     return options.inverse(this);
   });
+  handlebars.registerHelper('cellCanBePlayed', function (cellValue, readyToPlay, currentPlayerTurnToPlay, options) {
+    if (cellValue === "empty" && readyToPlay && currentPlayerTurnToPlay) {
+      return options.fn(this);
+    }
+    return options.inverse(this);
+  });
+  handlebars.registerHelper('falseOrFalse', function (value1, value2, options) {
```

Comment:
We could make it a less generic func, e.g needToWaitForTurn(isGameStarted,isMyTurn) ?